.nuget/packages.config

```diff
@@ -4,6 +4,7 @@
   <package id="DependDB.BuildProcessor.NuGetPreProcessor" version="2.0.0" />
   <package id="MSBuild.Extension.Pack" version="1.5.0" />
   <package id="NuGet.CommandLine" version="3.5.0" />
-  <package id="NUnit.Runners" version="2.6.3" />
+  <package id="NUnit.ConsoleRunner" version="3.11.1" targetFramework="net45" />
```

Comment:
> Also, all build.cmd files need to be upgraded, as Nunit3 requires a newer version of nuget.exe which the hardcoded link inside the build.cmd files cannot deliver.

Okay, switching to https://dist.nuget.org/win-x86-commandline/latest/nuget.exe is probably not an issue in build.cmd. Also, which version exactly is the minimum version? We will need to ensure that TeamCity also provides the correct minimum version when executing the build. Currently, we're using NuGet 4.5.1

Reply:
The error message was:
```
The 'NUnit 3.12.0' package requires NuGet client version '2.12' or above, but the current NuGet version is '2.8.60717.93'.
```

Meaning we should be fine

Reply:
yes.